protocols/directfund/directfund.go

```diff
@@ -404,18 +404,25 @@ func (o *Objective) clone() Objective {
 // IsDirectFundObjective inspects a objective id and returns true if the objective id is for a direct fund objective.
 func IsDirectFundObjective(id protocols.ObjectiveId) bool {
 	return strings.HasPrefix(string(id), ObjectivePrefix)
+
 }
 
 // ObjectiveRequest represents a request to create a new direct funding objective.
-type ObjectiveRequest struct {
+// There is no AppDefinition, since we currently only support full consensus rules for direct channels.
+type ObjectiveRequestWithoutAppDefinition struct {
 	CounterParty      types.Address
-	AppDefinition     types.Address
 	AppData           types.Bytes
```

Comment:
Should a `NullApp` request contain an `AppData` field? May be better to include `AppData` alongside `AppDefinition` in `ObjectiveRequest`.

Reply:
Resolved in [4909c90](https://github.com/statechannels/go-nitro/pull/804/commits/4909c9011f9480be1754ebf20634acd1fbd66230)